gallery/src/components/NxList/examples/NxListActions.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxButton, NxFontAwesomeIcon, NxList } from '@sonatype/react-shared-components';
+import { faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+function NxListActions() {
+  return (
+    <NxList>
+      <NxList.Item>
+        <NxList.Text>List item</NxList.Text>
+        <NxList.Actions>
+          <NxButton title="Edit" variant="icon-only"><NxFontAwesomeIcon icon={faEdit} /></NxButton>
+        </NxList.Actions>
+      </NxList.Item>
+      <NxList.Item>
+        <NxList.Text>List item deliberately made very long so that the text will wrap before it gets
```

Comment:
With multi-line block level elements we start their content on a new indented line.  We should see if there's an eslint setting for that.  It's tricky though since for inline elements on the other hand I'm fine with them being formatted in-line.
```suggestion
        <NxList.Text>
          List item deliberately made very long so that the text will wrap before it gets
```